### ISSUE TITLE
It is an SDK Project but seems not

### ISSUE THREAD

**Issue by VBAndCs** [paul1956/CSharpToVB#36]
This is the csprog file that is considered non-sdk in #26: 
```XML
<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <TargetFramework>netcoreapp3.1</TargetFramework>
    <RootNamespace>Microsoft.eShopWeb.Web</RootNamespace>
    <UserSecretsId>aspnet-Web2-1FA3F72E-E7E3-4360-9E49-1CCCD7FE85F7</UserSecretsId>
    <LangVersion>latest</LangVersion>
  </PropertyGroup>

  <ItemGroup>
    <Content Remove="compilerconfig.json" />
  </ItemGroup>


  <ItemGroup>
    <PackageReference Include="Ardalis.ListStartupServices" Version="1.1.3" />

    <PackageReference Include="MediatR" Version="7.0.0" />
    <PackageReference Include="MediatR.Extensions.Microsoft.DependencyInjection" Version="7.0.0" />
    <PackageReference Include="BuildBundlerMinifier" Version="2.9.406" Condition="'$(Configuration)'=='Release'" PrivateAssets="All" />
    <PackageReference Include="Microsoft.CodeAnalysis" Version="3.4.0" />
    <PackageReference Include="Microsoft.EntityFrameworkCore.InMemory" Version="3.1.0" />
    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="3.1.0" />
    <PackageReference Include="Microsoft.Web.LibraryManager.Build" Version="2.0.96" />
    <PackageReference Include="Swashbuckle.AspNetCore" Version="5.0.0-rc2" />
    <PackageReference Include="Swashbuckle.AspNetCore.SwaggerUI" Version="5.0.0-rc4" />
    <PackageReference Include="Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore" Version="3.1.0" />
    <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="3.1.0" />
    <PackageReference Include="Microsoft.AspNetCore.Identity.UI" Version="3.1.0" />
    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="3.1.0" />
    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="3.1.0">
      <PrivateAssets>all</PrivateAssets>
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
    </PackageReference>
  </ItemGroup>
  <ItemGroup>
    <Folder Include="wwwroot\fonts\" />
    <Folder Include="wwwroot\lib\" />
  </ItemGroup>
  <ItemGroup>
    <ProjectReference Include="..\ApplicationCore\ApplicationCore.csproj" />
    <ProjectReference Include="..\Infrastructure\Infrastructure.csproj" />
  </ItemGroup>
  <ItemGroup>
    <None Include="compilerconfig.json" />
    <None Include="wwwroot\images\products\1.png" />
    <None Include="wwwroot\images\products\10.png" />
    <None Include="wwwroot\images\products\11.png" />
    <None Include="wwwroot\images\products\12.png" />
    <None Include="wwwroot\images\products\2.png" />
    <None Include="wwwroot\images\products\3.png" />
    <None Include="wwwroot\images\products\4.png" />
    <None Include="wwwroot\images\products\5.png" />
    <None Include="wwwroot\images\products\6.png" />
    <None Include="wwwroot\images\products\7.png" />
    <None Include="wwwroot\images\products\8.png" />
    <None Include="wwwroot\images\products\9.png" />
  </ItemGroup>
  <ItemGroup>
    <Content Update="appsettings.json">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </Content>
    <Content Update="Views\Shared\Error.cshtml">
      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
    </Content>
    <Content Update="Views\Shared\_Layout.cshtml">
      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
    </Content>
    <Content Update="Views\Shared\_LoginPartial.cshtml">
      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
    </Content>
    <Content Update="Views\Shared\_ValidationScriptsPartial.cshtml">
      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
    </Content>
    <Content Update="Views\_ViewImports.cshtml">
      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
    </Content>
    <Content Update="Views\_ViewStart.cshtml">
      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
    </Content>
  </ItemGroup>
</Project>
```

It is an SDK, isn't it?
May be the problem comes from line termination. Some files use Lf termination, while others use CrLf. Try converting both and see if this is the problem.

**Comment by paul1956** [comment]
I was not supporting Web projects fixed in tomorrows build

**Comment by paul1956** [comment]
Fixed in 4.2.3.0
